### PR TITLE
Doc: added essence team documentation

### LINKED ISSUE #18: Write Essence way of working document
> Property (SEMAT): 
> 
> Assess and document (in one paragraph) your Team (p. 51 in the [Essence standard](https://www.omg.org/spec/Essence/1.2/PDF)v1.2) by evaluating the checklist on p. 52:
> In what state are you in? Why? What are obstacles to reach the next state?
> Assessment: The grader will read your statement and assess whether it corresponds to an honest and accurate self-evaluation.

This could be based on `ESSENCE.md` from the last assignment.

The document must be linked from `README.md`

### PR DESCRIPTION
**Pull request by hebbs4152**
## Describe your changes

Fixes #18 

## Checklist before requesting review

-   [x] Feature/fix PRs should add one atomic (as small as possible) feature or
        fix
-   [x] If your PR adds a feature or fix it also needs to add or modify at least
        one test
-   [x] All code in your PR needs to have been formatted
-   [x] The merge commit title should follow our commit prefix convention of
        either "feat", "fix", "chore", "doc", or "refactor"
-   [x] The merge commit body should reference at least one issue
-   [x] The code compiles and all the tests pass

### CODE REVIEW

**Comment by hebbs4152** [comment]
solves #18

File: README.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ The task is to create a continus integration server, the Canvas page can be foun
 
 `./gradlew run`
 
+## Essence - Way of Working
```
Comment (haval0) [inline README.md]:
```suggestion
## Essence - Team
```

File: Essence.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...our lunch meetings over discord call while also using the chat function to communicate throughout the day. Once each person assigned to an issue would finish a task they would then assign another team member to review and communication would shift to Github so we could keep a clear history of discussion. With the help of the clear communication and layout laid earlier, we continued through the *collaborating* state, continuing to work on the issues laid out in the states prior keeping clear communication, and are now in the *performing* state. We are working on meeting all of the commitments and addressing any last minute issues which may occur. We are making sure to work effectively and avoid backtracking by being aware of other people’s progress and keeping clear lines of communication. This state will continue until the presentation, at which point we will find out if there are additional aspects that need to be worked on. If so those will be managed as the initial issues were in the formed state by being assigned an issue and a person to complete it. After all the work is finished we will have reached the final *adjourned* state where there is no further work that must be done.
```
Comment (haval0) [inline Essence.md]: fix typo:
strukture -> structure

**Review by haval0 — CHANGES_REQUESTED** [review]
Covers all the bases, well done

I have two nitpicks:

**Review by danielwis — APPROVED** [review]

**Review by haval0 — APPROVED** [review]

**Pull request merged**